samples/javascript/E3_Monitor/index.js

```diff
@@ -0,0 +1,51 @@
+const df = require("durable-functions");
+const moment = require('moment');
+
+module.exports = df(function*(context) {
+    const input = context.df.getInput();
+    context.log("Received monitor request. location: " + (input ? input.location : undefined)
+        + ". phone: " + (input ? input.phone : undefined) + ".");
+
+    verifyRequest(input);
+
+    const endTime = moment.utc(context.df.currentUtcDateTime).add(6, 'h');
+    context.log("Instantiating monitor for " + input.location.city + ", " + input.location.state
+        + ". Expires: " + (endTime) + "."); // tostring in JavaScript?
```

Comment:
Do we want this comment in the official samples?

Reply:
No we don't. Good catch.